### PR TITLE
Setup

### LINKED ISSUE #24: Setup TravisCI

### PR DESCRIPTION
**Pull request by LHJE**
### Code Highlights:
- Recreates the Ruby on Rails app from scratch because there were too many errors with updating the old app.
- Adds a .travis.yml file to implement TravisCI
- Updates the Gemfiles to upgrade Rails to most recent  version
- Adds PR Template
### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.
### Issues:
* Closes #24 
* Closes #26 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion
- [x] Rubocop Violations: None

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+rbenv:
+- 2.5.3
+addons:
+  postgresql: 9.6
+script:
+- bundle exec rails db:{create,migrate} RAILS_ENV=test
+- bundle exec rspec
+deploy:
+  provider: heroku
+  # api_key:
```
Comment (leahriffell) [inline .travis.yml]: Nice @LHJE! Did the CLI command for getting this API key work for you?

Comment (LHJE) [reply .travis.yml]: It Did!  Thanks for helping with getting that implemented

**Review by LHJE — COMMENTED** [review]
We Nuked Everything on commit 57d311a because we were running into too many errors upgrading the rails app we'd started, and it wasn't too much work to just upgrade.

**Review by AngelaGuardia — APPROVED** [review]

**Review by dhalverson — APPROVED** [review]
Looks good, we worked on this together!

**Review by leahriffell — APPROVED** [review]
hooray

**Pull request merged**